src/NuGet.Core/NuGet.Build.Tasks.Pack/PackTask.cs

```diff
@@ -87,6 +91,22 @@ public override bool Execute()
         {
             try
             {
+                if(DebugPackTask)
```

Comment:
Do you want to leave this in for production?

Reply:
Yes, I think it will make it easier to debug in the future

Reply:
I'm not sure if this is a good idea. Anyways we would need to create debug bits to debug any customer issue.